neuraxle/distributed/streaming.py

```diff
@@ -27,9 +27,10 @@
 from threading import Thread
 from typing import Tuple, List, Union, Iterable
 
-from neuraxle.base import NamedTupleList, ExecutionContext, BaseStep, MetaStepMixin, NonFittableMixin, BaseSaver
+from neuraxle.base import NamedTupleList, ExecutionContext, BaseStep, MetaStep, BaseSaver, _FittableStep, \
+    BaseTransformer
 from neuraxle.data_container import DataContainer, ListDataContainer
-from neuraxle.pipeline import Pipeline, CustomPipelineMixin, MiniBatchSequentialPipeline, Joiner
+from neuraxle.pipeline import Pipeline, MiniBatchSequentialPipeline, Joiner
```

Comment:
This wasn't supposed to be in this pr, but the streaming steps had issues that I fixed here.

Reply:
ok